prometheus/graphite/bridge_test.go

```diff
@@ -291,7 +291,7 @@ func ExampleBridge() {
 	}
 
 	go func() {
-		// Start the service.
+		// Start something in a goroutine that uses metrics.
 		if err := http.ListenAndServe(":8080", nil); err != nil {
```

Comment:
What I meant is to not actually start an HTTP server here. (It suggests that starting an HTTP server is part of what you have to do to push Graphite metrics, while in fact that's not what you have to do anymore, in contrast to presenting Prometheus metrics in the usual way.)